src/styles/components/quota/styles.less

```diff
@@ -15,6 +15,10 @@
       margin-bottom: @base-spacing-unit;
     }
 
+    .unit {
+      font-weight: @quota-main-font-weight;
```

Comment:
we're introducing a `font-weight` of `400` here although our cards use `.unit-primary` which has a `font-weight` of `300` for all other components.

do we have good reason to introduce that special case here? /cc @mperrotti

Reply:
I would be against refactoring things that are not directly related.
I will change the font-weight here to be 300 for consistency.

Reply:
i think that's totally related. we are touching code that uses `.unit-primary`. unfortunately `.unit-primary` does not apply anything as it's out of scope, so we effectively introduced some dead code, which we now touch again. 

the perfect opportunity to either remove the dead code or make things better! :D

Reply:
I agree with Pierre's suggestion, but also agree that this should be non-blocking :)